appveyor.yml

```diff
@@ -10,16 +10,11 @@ init:
 - 'mkdir C:\ProgramData\PuppetLabs\puppet\var && exit 0'
 environment:
   matrix:
-  - PUPPET_GEM_VERSION: ~> 4.0
-    RUBY_VER: 21
-  - PUPPET_GEM_VERSION: ~> 4.0
-    RUBY_VER: 21-x64
-  - PUPPET_GEM_VERSION: ~> 5.0
-    RUBY_VER: 24
-  - PUPPET_GEM_VERSION: ~> 5.0
-    RUBY_VER: 24-x64
-  - PUPPET_GEM_VERSION: 4.7.1
-    RUBY_VER: 21-x64
+  - APPVEYOR_BUILD_WORKER_CLOUD: puppet-appveyor-test
+    PUPPET_GEM_VERSION: ~> 5.0
+    APPVEYOR_BUILD_WORKER_IMAGE: 2012R2 PS4
+    TEST_VARIABLE: 42
+    RUBY_VERSION: 24    
```

Comment:
Doesn't look like it picked up Ruby 2.4... I think there's some magic baked into the AppVeyor default images that use this environment variable to set current ruby version via URU or similar.

Reply:
The gem install failure is for this gem - https://rubygems.org/search?utf8=%E2%9C%93&query=puppet-module-win-default

Note that only 2.1, 2.3 and 2.4 versions are published.

Reply:
Not really magic.  First line in the file
https://github.com/puppetlabs/puppetlabs-powershell/blob/master/appveyor.yml#L25-L26

Note `RUBY_VER` has been renamed `RUBY_VERSION`

Reply:
Thanks @glennsarti That worked a treat - and checks pass.